README.md

```diff
@@ -34,10 +34,11 @@ Easy Model Deployer is a lightweight tool designed to simplify the deployment of
 
 ## 🚀 Quick Install
 
-Install EMD with `pip`, currently support for Python 3.9 and above:
+Install Easy Model Deployer with `pip`, currently support for Python 3.9 and above:
 
 ```bash
-pip install https://github.com/aws-samples/easy-model-deployer/releases/download/emd-0.7.1/emd-0.7.1-py3-none-any.whl
```

Comment:
can we add instruction for intallation from latest main? like pip install git+https://github.com/aws-samples/easy-model-deployer.git

Reply:
IMO, It's a bit unstable for the main branch than release version, if this is a not recommended installation method, I prefer do not add in the readme, two entrances might confuse some user.